package.json

```diff
@@ -5,9 +5,10 @@
   "dependencies": {
     "@ant-design/icons": "4.8.1",
     "@ckb-lumos/base": "0.22.2",
+    "@ckb-lumos/codec": "0.22.2",
     "@ckb-lumos/config-manager": "0.22.2",
     "@ckb-lumos/helpers": "0.22.2",
-    "@ckb-lumos/lumos": "^0.22.2",
```

Comment:
Why remove `@ckb-lumos/lumos`? I thought most functions can be imported from the top package `@ckb-lumos/lumos`

Reply:
I thought `@ckb-lumos/lumos` is a collection of other `lumos` packages that will bring in all the sub-packages, some of which are brought in with some version dependency issues. functions currently imported from `@ckb-lumos/lumos` can also be import in directly from sub-packages, so i remove this top-level package